ext/fetch/22_body.js

```diff
@@ -64,17 +64,21 @@
   }
 
   class InnerBody {
+    #contentLength = null;
+
     /**
      * @param {ReadableStream<Uint8Array> | { body: Uint8Array | string, consumed: boolean }} stream
      */
-    constructor(stream) {
+    constructor(stream, { contentLength } = {}) {
```

Comment:
nit: avoid objects (perf reasons) since this is internal API
```suggestion
    constructor(stream, contentLength) {
```